src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -2830,20 +2727,7 @@ internal static void SetConsoleTextAttribute(ConsoleHandle consoleHandle, WORD a
         private static uint CodePageToCharSet(uint codePage)
         {
             const uint OEM_CHARSET = 255;
-            // Suppress the PreFAST warning about not using Marshal.GetLastWin32Error() to
-            // get the error code.
-#pragma warning disable 56523
-#if CORECLR // TranslateCharsetInfo exists in an extension API set 'ext-ms-win-gdi-font-l1-1-1.dll', which is not available in NanoServer.
             return OEM_CHARSET;
-#else
-            CHARSETINFO csi;
-            const DWORD TCI_SRCCODEPAGE = 2;
-            if (!NativeMethods.TranslateCharsetInfo((IntPtr)codePage, out csi, TCI_SRCCODEPAGE))
```

Comment:
We should remove all unused NativeMethods declarations (TranslateCharsetInfo, GetConsoleWindow, etc.).

Reply:
Will remove